mult_div.java

```diff
@@ -14,9 +14,11 @@ public static void main(String[] args) {
 		b = sc.nextInt();
 		int mult_ans = a*b;
 		// division code to be added
+		int div_ans = a/b;
```

Comment:
Should have used float datatype for division operation.

Reply:
Ok will use float